src/components/Profile/Profile/FeatureContainer/DoctorsFeature/OrderManagement/OrderManagement.tsx

```diff
@@ -11,7 +11,7 @@ const WorkTimeManagement: React.FC = () => {
       <ContainersName>Patient</ContainersName>
 
       <ContainerContent>
-        {!workTime?.length ? (
+        {!workTime?.filter((appointment) => appointment.patientName).length ? (
```

Comment:
maybe better to move into separate const?

Reply:
what do you mean here?